src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -353,33 +373,33 @@ private void incomingPacketInfo(
 
             long[] deltas = new long[]{tsDelta, tDelta, sizeDelta};
```

Comment:
I believe after passing arrays as params to the computeDelta function  (the computeDelta function modifies that array), the only way to retrieve the modified values of the arrays is to get them by referencing the array i.e `deltas[0]` since arrays are objects & on the heap. Using `tsDelta` directly will give us the value of `tsDelta `initially set since `tsDelta `is of primitive type on the stack, and not scoped within computeDelta function. In the cpp code, the values where passed & retrieved by reference i.e `&tsDelta`. Let me know if there is any other way. Thanks.